lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -18,9 +17,37 @@ description:
 destroy:
   config_set: "no vrf context <vrf>"
 
+feature_vn_segment_vlan_based:
+  _exclude:
+    - /N7K/  # Support for multi-tenancy full only
```

Comment:
Have you tested this diffset on an N7K ?

If you don't have an N7K testbed then as a test you can try changing that regexp above to N3K or N9K just to see if the vn_segment config_get_token gets called.

The rest of your changes look fine.